configurations/RX-woo/sources/Source-RX Fundament Tilburg-v0.0.1.json

```diff
@@ -0,0 +1,55 @@
+{
+	"@context": {
+		"schema": "http:\/\/schema.org",
+		"register": "Not Implemented"
+	},
+	"@type": "source",
+	"@id": "https://tilburg.accept.commonground.nu/Source-RX Fundament Tilburg.json",
+	"id": 1,
+	"uuid": "12f23206-a455-42b2-a717-15dbad24ff5e",
```

Comment:
Shouldn't these id's be cleaned from export?

Reply:
UploadService should filter this